src/app/lib/utilities/getRemoteConfig/getRemoteConfigWithCache.js

```diff
@@ -0,0 +1,11 @@
+import Cache from 'lru-cache'; // make a note of why we would choose this and give alternatives.
+import getRemoteConfig from '.';
+
+const cacheMaxItems = parseInt(process.env.SIMORGH_CONFIG_CACHE_ITEMS, 10); // can this library / is there an alternative that lets us limit the size of the cache too?
+const cacheMaxAge = parseInt(process.env.SIMORGH_CONFIG_CACHE_MAX_AGE, 10); // seconds - could make this different in dev vs production
```

Comment:
Could we rename the env variable so that this comment is no longer needed e.g. `SIMORGH_CONFIG_CACHE_MAX_AGE_SECONDS`

Reply:
Good point! Updated